decentraland/components/presentational/AssetName.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { StyleSheet, View, Text } from "react-native";
+import PropTypes from "prop-types";
+import { responsiveHeight } from "react-native-responsive-dimensions";
+import Colors from "@constants/Colors";
+
+export function AssetName({ asset }) {
```

Comment:
Does this address https://github.com/tasitlabs/tasit/issues/261 ?

Reply:
Yep. Added to the PR description.

Reply:
Thanks. Resolving this.